README.md

```diff
@@ -189,26 +184,29 @@ const { flags, values, positionals } = parseArgs({ args, options });
   - `"0o22"`
 - Does it coerce types?
   - no
-- Does `--no-foo` coerce to `--foo=false`?  For all flags?  Only boolean flags?
-  - no, it sets `{args:{'no-foo': true}}`
+- Does `--no-foo` coerce to `--foo=false`?  For all options?  Only boolean options?
+  - no, it sets `{values:{'no-foo': true}}`
 - Is `--foo` the same as `--foo=true`?  Only for known booleans?  Only at the end?
-  - no, `--foo` is the same as `--foo=`
+  - no, `--foo` is a boolean option and `--foo=true` is a string option
```

Comment:
when `foo` is configured as a boolean option and strict true, what does `--foo=true` do?

Reply:
In answer to your question: we are planning it throws. Prompted by your question... 

Hmm, my change was not good as a general description. (I had a particular configuration and issue in mind at the time.)

How about less technical but hopefully more helpful:

> no, they are not the same. There is no special handling of `true` as a value so it is just another string.

Reply:
Done. Thanks.